src/Server/Workspace.php

```diff
@@ -200,4 +214,75 @@ public function xdependencies(): array
         }
         return $dependencyReferences;
     }
+
+    /**
+     * Fires when client changes settings in the client
+     *
+     * The default paramter type is Options but it also accepts different types
+     * which will be transformed on demand.
+     *
+     * Currently only the vscode format is supported
+     *
+     * @param mixed|null $settings
+     * @return bool
```

Comment:
I know and I was hesitating to add the return value, but for some tests I need them.
When you have another idea how I can test them, tell me.

see for example `testNoOptionPassed` or `testNoChangedOptions`